docs/src/SUMMARY.md

```diff
@@ -0,0 +1,19 @@
+# Summary
+
+- [Introduction](./introduction.md)
+- [Install](./install.md)
```

Comment:
should we drop this?

Reply:
it's discardable 100% but we should at very least link oranda's own generated stuff *from* the book with the heading "install" so people can find it quickly if they end up here

Reply:
we'll handle book updates post-release, since this is not a release we are formally announcing